cmd/ndm_daemonset/controller/blockdevice.go

```diff
@@ -85,7 +85,14 @@ func (di *DeviceInfo) getObjectMeta() metav1.ObjectMeta {
 		Annotations: make(map[string]string),
 		Name:        di.UUID,
 	}
-	objectMeta.Labels[KubernetesHostNameLabel] = di.NodeAttributes[HostNameKey]
+	//objectMeta.Labels[KubernetesHostNameLabel] = di.NodeAttributes[HostNameKey]
+	for k, v := range di.NodeAttributes {
```

Comment:
Currently in `NodeAttributes`, do we have anything other than hostname/node name?

Reply:
No.